src/source/lexbase.d

```diff
@@ -144,7 +144,7 @@ mixin template LexBaseImpl(Token, alias BaseMap, alias KeywordMap,
 		return t.type == TokenType.End;
 	}
 
-private:
+package(source):
```

Comment:
Is it possible to increase visibility more granularly? I don't see why `reachedOEF` couldn't be public, while, on the other hand, many things in LexBaseUtils shouldn't be, as well as everything else that's defined in there.

Reply:
done